### PR TITLE
feat: pk_sequence reset added to csv_load.rake

### PR DESCRIPTION
**Pull request by Penitent0**
Added pk_sequence reset to csv_load.rake

### CODE REVIEW

File: lib/tasks/csv_load.rake [hunk 1]
```diff
@@ -90,6 +90,14 @@ namespace :csv_load do
     end
   end
 
+  desc "Scorched Earth"
```
Comment (aleish-m) [inline lib/tasks/csv_load.rake:93]: Fun, for now, might need to refactor later for good naming hygiene.

File: lib/tasks/csv_load.rake [hunk 2]
```diff
@@ -90,6 +90,14 @@ namespace :csv_load do
     end
   end
 
+  desc "Scorched Earth" 
+  task :pk_reset => :environment do
+    ActiveRecord::Base.connection.tables.each do |t|
```
Comment (eriggs0207) [inline lib/tasks/csv_load.rake:95]: Great way to reset the pk.  May have to change the name from scorched earth to somthing that follows convention

**Review by eriggs0207 — APPROVED** [review]

File: lib/tasks/csv_load.rake [hunk 3]
```diff
@@ -90,6 +90,14 @@ namespace :csv_load do
...
+      ActiveRecord::Base.connection.reset_pk_sequence!(t)
+    end
+  end
+
   desc "csv load all"
-  task :all => [:customers, :invoice_items, :invoices, :items, :merchants, :transactions]
-end
\ No newline at end of file
+  task :all => [:customers, :invoices, :merchants, :items, :transactions, :invoice_items, :pk_reset]
+end
```
Comment (aleish-m) [inline lib/tasks/csv_load.rake:102]: This is super awesome that this works thank you for figuring this out and working through it.

**Review by aleish-m — APPROVED** [review]

**Comment by aleish-m** [comment]
[database setup ](https://trello.com/c/NkagsXjT/41-database-setup)

**Pull request merged**